pkg/certificate/providers/vault/certificate_manager_test.go

```diff
@@ -52,6 +53,7 @@ var _ = Describe("Test client helpers", func() {
 })
 
 func TestNew(t *testing.T) {
+	tassert := assert.New(t)
```

Comment:
Let's move this into the `t.Run` so the error message shows exactly which test case failed. As-is the error message looks like this:
```
--- FAIL: TestNew (0.01s)
    certificate_manager_test.go:107:
                Error Trace:    certificate_manager_test.go:107
                Error:          An error is expected but got nil.
                Test:           TestNew
                Messages:       expected error, got nil
```

But with that change it would be:
```
--- FAIL: TestNew (0.01s)
    --- FAIL: TestNew/valid_inputs (0.00s)
        certificate_manager_test.go:107:
                Error Trace:    certificate_manager_test.go:107
                Error:          An error is expected but got nil.
                Test:           TestNew/valid_inputs
                Messages:       expected error, got nil
```